analytics/views/stats.py

```diff
@@ -406,6 +418,7 @@ def get_chart_data(
         InstallationCount: -1,
         RealmCount: realm.id,
         UserCount: user_profile.id,
+        StreamCount: int(stream_id or 0),
```

Comment:
What's the purpose of the `or 0` here?

Reply:
I see, it's to avoid an exception, fixed this to follow the style used for similar issues just below.